src/Data/ClientId.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Data;
+
+/**
+ * Class ClientId
+ * @package ILIAS\Data
+ * @author Michael Jansen <mjansen@databay.de>
+ */
+class ClientId
+{
+	/**
+	 * @var string
+	 */
+	private $clientId;
+
+	/**
+	 * ClientId constructor.
+	 * @param $clientId
+	 */
+	public function __construct($clientId)
+	{
+		if (!is_string($clientId)) {
```

Comment:
Couldn't we just typehint this?

Reply:
Yes, we could. I will changed this tomorrow.